src/service/gnuradio/GnuRadioAcquisitionWorker.hpp

```diff
@@ -513,7 +513,11 @@ class GnuRadioAcquisitionWorker : public Worker<serviceName, TimeDomainContext,
                     reply.triggerIndices.push_back(idx);
                     reply.triggerEventNames.push_back(std::get<std::string>(tagMap.at(gr::tag::TRIGGER_NAME)));
                     reply.triggerTimestamps.push_back(static_cast<int64_t>(std::get<uint64_t>(tagMap.at(gr::tag::TRIGGER_TIME))));
-                    reply.triggerOffsets.push_back(std::get<float>(tagMap.at(gr::tag::TRIGGER_OFFSET)));
+                    if (tagMap.contains(gr::tag::TRIGGER_OFFSET)) {
```

Comment:
Maybe use find instead of double-lookup?

Reply:
Good point, although then it should probably also be changed in L506 for consistency and there it's trickier to get the short circuit right and keeping the code readable. I'll add this to the list of things to do in the follow-up.